config/foreman.hiera/custom.yaml

```diff
@@ -24,3 +24,9 @@
 # apache::mod::prefork::serverlimit: 256
 # apache::mod::prefork::maxclients: 256
 # apache::mod::prefork::maxrequestsperchild: 4000
+#
+# Here are some examples of how you tune the Postgres options if needed:
```

Comment:
I think the official spelling is PostgreSQL. @evgeni has been persistent in pointing this out so let's keep it correct.